server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/SearchAction.java

```diff
@@ -69,29 +71,32 @@ public void define(NewController context) {
       .addFieldsParam(ALL_FIELDS)
       .addPagingParams(100, MAX_LIMIT)
       .addSearchQuery("sonar-users", "names");
+
+    action.createParam(PARAM_ORGANIZATION_KEY)
+      .setDescription("Key of organization. If not set then groups are searched in default organization.")
+      .setExampleValue("my-org")
+      .setSince("6.2");
   }
 
   @Override
   public void handle(Request request, Response response) throws Exception {
     userSession.checkLoggedIn();
+
     int page = request.mandatoryParamAsInt(Param.PAGE);
     int pageSize = request.mandatoryParamAsInt(Param.PAGE_SIZE);
     SearchOptions options = new SearchOptions()
       .setPage(page, pageSize);
 
-    String query = StringUtils.defaultIfBlank(request.param(Param.TEXT_QUERY), "");
+    String query = defaultIfBlank(request.param(Param.TEXT_QUERY), "");
```

Comment:
I know it's legacy code, but looking at methods consuming variable `query`, they all accept `null` which means no filtering on name.

Providing an empty string actually adds a where clause `and upper(g.name) like '%%' escape '/'` to SQL requests (in addition to calling lots of string manipulating code for nothing) and it's definitely a bad idea performance wise